redisson/src/main/java/org/redisson/connection/pool/ConnectionPool.java

```diff
@@ -44,7 +44,7 @@
  */
 abstract class ConnectionPool<T extends RedisConnection> {
 
-    private final Logger log = LoggerFactory.getLogger(getClass());
+    private static final Logger log = LoggerFactory.getLogger(ConnectionPool.class);
```

Comment:
this needs to remain unchanged or be added to all inheritance classes with concrete class, not ConnectionPool.